src/WatcherCommand.php

```diff
@@ -38,9 +41,26 @@ protected function determineOptions(InputInterface $input): array
         return $options;
     }
 
+    protected function getOptionsConfigFile()
+    {
+        $configPath = getcwd();
+        $configFile = $configPath . self::PHPUNIT_WATCH_CONFIG_FILENAME;
+
+        while (! file_exists($configFile)):
+            $configPath = dirname($configPath);
+            $configFile = $configPath . self::PHPUNIT_WATCH_CONFIG_FILENAME;
+            if ($configPath === '/') {
+                $configFile = self::PHPUNIT_WATCH_CONFIG_FILENAME;
+                break;
+            }
+        endwhile;
+
+        return $configFile;
+    }
+
     protected function getOptionsFromConfigFile(): array
     {
-        $configFile = getcwd().'/.phpunit-watcher.yml';
+        $configFile = $this->getOptionsConfigFile();
```

Comment:
The code below this line will parse the file. Of course that should be done in the `getOptionsFromFile` now.